content/docs/user-guide/components/reference/physx/shape-collider.md

```diff
@@ -1,92 +1,78 @@
+| **Trigger** | Make this collider a trigger. A trigger performs a quick overlap test and does not apply forces or return contact point information. Use this to speed-up PhysX computations where a simple overlap between colliders is sufficient. Triangle meshes are not supported as triggers. | Boolean | `On` |
+| **Simulated** | Include this collider in the physics simulation. | Boolean | `On` |
+| **In Scene Queries** | Allow this collider can be queried for raycasts, shapecasts, and overlap. | Boolean | `On` |
+| **Library (Physics Materials)** | The physics material library of the project. | `.physmaterial` | The global `.physmaterial` asset |
+| **Slots (Physics Materials)** | Choose a physics material for each material of this collider. Physics materials define physical properties for the surface such as dynamic and static friction, and density. A collider can have multiple physics materials assigned. | Physics material from the assigned `.physmaterial` | `<Default Physics Material>` |
+| **Tag** | Set a tag for this collider. Tags can be used to quickly identify components in script or code. | String |  |
+| **Rest offset** | PhysX bodies come to rest separated by the sum of their **Rest offset** values. The **Rest offset** value must be less than the **Contact offset** value. | `-Infinity` to `50.0` | `0.0` |
+| **Contact offset** | PhysX bodies generate contacts when they are within the sum of their **Contact offset** values. The **Contact offset** value must be greater than the **Rest offset** value. | `0.0` to `50.0` | `0.02` |
+| **Draw Collider** | Render the collider in the viewport. | Boolean | `On` |
 
 ## Complex polygon prism shapes 
 
 The [Polygon Prism Shape](/docs/user-guide/components/reference/shape/polygon-prism-shape/) is automatically subdivided into convex portions, which means that polygon prisms can be used with dynamic rigid bodies or as triggers in PhysX. The subdivision is automatically updated if the vertices of the polygon prism are modified.
 
-![\[A complex polygon prism can't be converted to convex geometry.\]](/images/user-guide/component/physx/physx/ui-physx-shape-collider-B.png)
+![A complex polygon prism can't be converted to convex geometry.](/images/user-guide/components/reference/physx/ui-shape-collider-b.png)
 
-If the vertices are modified so that the polygon prism is no longer a simple polygon, for example, if the polygon prism is self-intersecting, it isn't possible to subdivide the polygon prism into convex pieces. An error will display in the Editor Console, as shown in the following example.
+If the vertices are modified so that the polygon prism is no longer a simple polygon, it isn't possible to subdivide the polygon prism into convex pieces. In the example image above, the polygon prism is self-intersecting. If the polygon prism can't be subdivided into convex pieces, nn error will display in the Editor Console, as shown in the following example.
```

Comment:
```suggestion
If the vertices are modified so that the polygon prism is no longer a simple polygon, it isn't possible to subdivide the polygon prism into convex pieces. In the example image above, the polygon prism is self-intersecting. If the polygon prism can't be subdivided into convex pieces, an error will display in the Editor Console, as shown in the following example.
```